cmd/internal/flowmode/cmd_run.go

```diff
@@ -95,18 +96,21 @@ depending on the nature of the reload error.
 		StringVar(&r.clusterJoinAddr, "cluster.join-addresses", r.clusterJoinAddr, "Comma-separated list of addresses to join the cluster at")
 	cmd.Flags().
 		BoolVar(&r.disableReporting, "disable-reporting", r.disableReporting, "Disable reporting of enabled components to Grafana.")
+	cmd.Flags().
+		BoolVar(&r.enableGoProfiling, "enable-go-profiling", r.enableGoProfiling, "Enable /debug/pprof profiling endpoints.")
```

Comment:
I'd like to group this under `server.http.`; can we call this `server.http.enable-pprof`?

Reply:
Done